### ISSUE TITLE
total_retention_in_days param not accepting values greater than 730

### ISSUE THREAD

**Issue by leuthelt** [hashicorp/terraform-provider-azurerm#24860]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment and review the [contribution guide](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/README.md) to help.

### Terraform Version

1.7.1

### AzureRM Provider Version

3.91.0

### Affected Resource(s)/Data Source(s)

azurerm_log_analytics_workspace_table

### Terraform Configuration Files

```hcl
resource "azurerm_log_analytics_workspace_table" "activity" {
  workspace_id            = azurerm_log_analytics_workspace.law.id
  name                    = "AzureActivity"
  plan                    = "Analytics"
  retention_in_days       = 730
  total_retention_in_days = 1095
}
```

### Debug Output/Panic Output

```shell
Error: expected total_retention_in_days to be in the range (30 - 730), got 1095
```

### Expected Behaviour

Like described in https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/log_analytics_workspace_table, I expect correct validation of total_retention_in_days param, which should allow numbers from 30 and 4383.

### Actual Behaviour

Error: expected total_retention_in_days to be in the range (30 - 730), got 1095

**Comment by mikemadeja** [comment]
Hello, I created a PR to correct the number.

https://github.com/hashicorp/terraform-provider-azurerm/pull/24900

**Comment by rcskosir** [comment]
Thanks for taking the time to submit this issue. It looks like this has been resolved as of #24900. As such, I am going to mark this issue as closed. If that is not the case, please provide additional information including the version in which you are still experiencing this issue, thanks!
